src/variableDiagnosticsProvider.ts

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+import { workspace, languages, Diagnostic, DiagnosticSeverity, DiagnosticCollection, TextDocument, Range, Position, Disposable } from 'vscode';
+
+import { event } from "./events/requestVariableEvent";
```

Comment:
How about doing a import rename like `import { event as OnRequestVariableEvent } from "./events/requestVariableEvent";` I think it's more readable than `event`

Reply:
Sure. I'll rename it.